hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/process/RunningSubscriptionStatus.java

```diff
@@ -0,0 +1,28 @@
+package pl.allegro.tech.hermes.consumers.supervisor.process;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.codehaus.jackson.annotate.JsonCreator;
+
+import java.util.Map;
+
+public class RunningSubscriptionStatus {
+    private final String qualifiedName;
+
+    private final Map<Signal.SignalType, Long> signalTimesheet;
+
+    @JsonCreator
```

Comment:
At the moment this is used to parse signals in our assertions. This can be done outside of this class but seems relatively harmless to me.